demisto_sdk/common/hook_validations/base_validator.py

```diff
@@ -100,3 +101,18 @@ def is_subset_dictionary(new_dict, old_dict):
             if arg not in old_dict.keys() and required:
                 return False
         return True
+
+    def _is_id_equals_name(self, file_type):
```

Comment:
jsons don't get here, right?

Reply:
right. this function is only called from script, integration or playbook validator